lib/utils.js

```diff
@@ -1,3 +1,5 @@
+const _ = require('lodash');
+
 // this will have non-OAS-related utils
 
 module.exports = {
```

Comment:
instead of separate handling everywhere in the switch case, if we could do a top-level lower casing once it would clean the code